staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -298,7 +299,12 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 		}
 
 		if err := r.watchHandler(w, &resourceVersion, resyncerrc, stopCh); err != nil {
-			if err != errorStopRequested {
+			switch err {
+			case errorStopRequested:
+				// no extra log message
+			case ErrStaleWatch:
+				klog.V(1).Infof("%s: watch of %v ended with: %v", r.name, r.expectedType, err)
```

Comment:
would even go higher with the verbosity level. 4 maybe. So it is hidden normally.

Reply:
done, made it a 4